### PR TITLE
Fix broken links in DEP 1

### PR DESCRIPTION
**Pull request by michael-k**
Ref https://github.com/django/django/commit/3d14cbc86781ea1051af7f0c421bee3ecf2f9842

### CODE REVIEW

File: final/0001-dep-process.rst [hunk 1]
```diff
@@ -71,15 +71,14 @@ The Technical Board
...
+    membership https://www.djangoproject.com/foundation/teams
+    /#technical-board-team>`_.
 
 Core Developers
     Similarly, there are several references to **Core Developers** (sometimes
     "core devs"). This refers to the members of Django's `core team
     <https://docs.djangoproject.com/en/dev/internals/organization/#core-team>`_,
-    and specifically those with `commit access
-    <https://docs.djangoproject.com/en/dev/internals/roles/#committers>`_.
+    and specifically those with commit access.
```
Comment (michael-k) [inline final/0001-dep-process.rst]: I haven't found a suitable replacement for this.  But the previously linked text did not provide any additional information anyway:

```
Committers
==========

Most :ref:`core team <core-team>` members have commit access. They're called
"committers" or "core developers".

Being part of the core team is a pre-requisite for having commit access.
```

Comment (adamchainz) [reply final/0001-dep-process.rst]: Seems fine to me.

**Comment by adamchainz** [comment]
Thank you! 👍

File: final/0001-dep-process.rst [hunk 2]
```diff
@@ -71,15 +71,14 @@ The Technical Board
     <https://docs.djangoproject.com/en/dev/internals/organization/#technical-
     board>`_, the group of experienced and active committers who steer technical
     choices. Django's documentation lists `the current Technical Board
-    membership <https://docs.djangoproject.com/en/dev/internals/roles
-    /#technical-board>`_.
+    membership https://www.djangoproject.com/foundation/teams
```
Comment (adamchainz) [inline final/0001-dep-process.rst]: Oh golly, the opening `<` went missing - fixed in #58

**Pull request merged**